source/mainnet/net/web3-id/index.rst

```diff
@@ -0,0 +1,107 @@
+
+As an alternative to the Concordium Web3 ID Issuer Front End, Concordium additionally provides an `issuer service <https://github.com/Concordium/concordium-web3id/tree/main/services/web3id-issuer>`_ that is designed to handle all of the chain-specific interactions, and provides a simple REST API, so that only the custom business logic of the issuer must be implemented. Note that the service must be run by the issuer themselves, since only they have the secret keys necessary to run it.
+
+Regardless of whether you use the Concordium Web3 ID Issuer Front End or the issuer service, the issuer must provide JSON schemas for their credentials, and metadata. These schemas and metadata are used by the holder's wallet to display the credentials in the wallet.
+
+Verifier
+--------
+
+A verifier is expected to be a business or another use-case that will only provide a service provided specific proofs can be provided by the holder, for example proof of ownership of specific credential, such as a concert ticket.
+This will typically consist of two components.
+
+1. A dApp that will interact with the wallet to request from the user to attest to specific properties of their credentials. The result of such a request is a :ref:`verifiable presentation<glossary-verifiable-presentation>`.
+2. A back-end component that will verify the provided verifiable presentation and allow the desired action or not.
+
+Verification of presentations has two components, the cryptographic verification of the zero-knowledge proofs contained within, which establish that the holder indeed owns the relevant credentials and they are issued by the stated issuer, and the business logic checking that the required property was attested.
+In particular, a key part of the business logic is whether the verifier trusts a particular issuer, which is identified by a smart contract address and the public key registered therein.
+
+Concordium runs an instance of the verifier for `Mainnet <https://web3id-verifier.mainnet.concordium.software/v0/verify>`_ or `Testnet <https://web3id-verifier.testnet.concordium.com/v0/verify>`__. Using Concordium hosted services means that the verifier places trust in Concordium, but simplifies the implementation of the verifier. You still have to provide a user interface or front end (usually a dApp) that calls the Concordium hosted verifier. Otherwise, if you want to run the verification service yourself, Concordium simplifies the checking of the cryptographic part by providing a `verifier service <https://github.com/Concordium/concordium-web3id/tree/main/services/web3id-verifier>`_. To read more about the verification service, see :ref:`Tool to verify credentials<verifier-tool>`.
+
+An example verifier dApp can be found (link), and an example back end can be found (link).
```

Comment:
Can we link to Concordia here?

The example verifier backend source code is at https://github.com/Concordium/concordium-web3id/tree/main/examples/some-verifier